src/ngx_http_lua_log.c

```diff
@@ -428,6 +428,30 @@ ngx_http_lua_ffi_errlog_get_sys_filter_level(ngx_http_request_t *r)
     return log_level;
 }
 
+
+int
+ngx_http_lua_ffi_raw_log(ngx_http_request_t *r, int level, u_char *s,
```

Comment:
Maybe it's better to use `unsigned int` here instead of `int` so that we do not have to do the explicit type cast to `ngx_uint_t` below?

Reply:
SGTM, changed.